### PR TITLE
feat: change cursor when panning

### PR DESCRIPTION
**Pull request by judicaelandria**
https://user-images.githubusercontent.com/46365844/187202870-f01d6640-bec5-4cb3-958c-47363416b5f7.mov

### CODE REVIEW

**Comment by judicaelandria** [comment]
Close https://github.com/tldraw/tldraw/issues/487

**Comment by alex-kim-dev** [comment]
Hi! It would be great if this also worked for middle click panning, which is what I'm used to, thanks!

File: packages/tldraw/src/Tldraw.tsx [hunk 1]
```diff
@@ -504,117 +507,141 @@ const InnerTldraw = React.memo(function InnerTldraw({
...
-                  <StyledSpacer />
-                  {showTools && !readOnly && <ToolsPanel />}
-                </>
-              )}
-            </StyledUI>
-          )}
-        </StyledLayout>
-      </IntlProvider>
-    </ContainerContext.Provider>
+    <Wrapper
```
Comment (steveruizok) [inline packages/tldraw/src/Tldraw.tsx:526]: Could you get rid of the Wrapper component and move these props to StyledLayout?

**Comment by steveruizok** [comment]
I've switched this out to something a little more manual that doesn't add any new HTML elements.

**Pull request merged**